docs/dxp/latest/en/content-authoring-and-management/content-dashboard/content-dashboard-interface.md

```diff
@@ -101,18 +101,21 @@ In the list of contents, you can find the following columns:
 | Title | Article title(*). |
 | Author | Author's name. |
 | Type | Specifies a [Web Content Article](../web-content/web-content-articles/adding-a-basic-web-content-article.md) or [Documents](../documents-and-media/documents-and-media-overview.md) asset type (available in Liferay DXP 7.4+.) |
-| Subtype | Article's [Subcategory](../tags-and-categories/defining-categories-and-vocabularies-for-content.md#creating-subcategories). |
+| Subtype | Content [Subcategory](../tags-and-categories/defining-categories-and-vocabularies-for-content.md#creating-subcategories). |
 | Site or Asset Library | Location of the article in the Liferay DXP instance. |
 | Status | Article workflow status. Articles with more than one version in different statuses show all than statuses. |
 | *First Vocabulary* | This is the first Vocabulary in the Content Audit Tool. |
 | *Second Vocabulary* | The optional second Vocabulary in the Content Audit Tool. |
 | Modified Date | Date or time of the last article update. |
 
-(*) The *Page* (![Page](../../images/icon-page.png)) icon denotes articles based on a [Display Page Template](../../site-building/displaying-content/using-display-page-templates/about-display-page-templates-and-display-pages.md).
+(*) The *Page* (![Page](../../images/icon-page.png)) icon denotes content based on a [Display Page Template](../../site-building/displaying-content/using-display-page-templates/about-display-page-templates-and-display-pages.md).
 
 ```{note}
 The *First Vocabulary* and *Second Vocabulary* columns depend on the Vocabularies you select.
 ```
+```{warning}
+The *subtypes* column only displays Basic documents (doc, pdf, xls, jpg, etc.) and External Video Shortcuts on the content performance panel.
```

Comment:
Hi @Fabiomorais87 ! Sorry, I think I didn't explain myself correctly or I'm not understanding it right. It is not that the subtype column only shows these subtypes, we are only displaying basic documents and external video shortcuts in the content dashboard table. For me it would make more sense something like "_The subtypes of documents displayed in the content dashboard are Basic documents (doc, pdf, xls, jpg, etc.) and External Video Shortcuts_". My english is not perfect and I might be understanding wrong 😅 

Everything else looks very good 👍